recipes/pymatgen/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set name = "pymatgen" %}
+{% set version = "4.5.7" %}
+{% set sha256 = "3793aaf8224897f0f4d8fd6ab1a8ffcc40cbd5e89299960f4ff0a07bd75af727" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win and py<35]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pmg = pymatgen.cli.pmg:main
+    - feff_input_generation = pymatgen.cli.feff_input_generation:main
+    - feff_plot_cross_section = pymatgen.cli.feff_plot_cross_section:main
+    - feff_plot_dos = pymatgen.cli.feff_plot_dos:main
+    - gaussian_analyzer = pymatgen.cli.gaussian_analyzer:main
+    - get_environment = pymatgen.cli.get_environment:main
+    - pydii = pymatgen.cli.pydii:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.9
+    - toolchain
+  run:
+    - python
+    - numpy >=1.9
+    - six
+    - requests
+    - pyyaml >=3.11
+    - monty >=0.9.6
+    - scipy >=0.14
+    - pydispatcher >=2.0.5
+    - tabulate
+    - spglib >=1.9.8.7
+    - matplotlib >=1.5
+    - palettable >=2.1.1
+    - enum34  # [py27]
+
+test:
+  commands:
+  - pmg --help
+  - gaussian_analyzer --help
```

Comment:
Please fix the spacing in this list section so that it is indented 2 more spaces.